scripts/build-controller-image.sh

```diff
@@ -17,43 +16,27 @@ check_is_installed docker
 
 USAGE="
 Usage:
-  $(basename "$0") [-q] [-s <AWS_SERVICE>] [-i <Docker image tag>]
+  $(basename "$0") <aws_service>
 
-Builds the Docker image for an ACK service controller.
+Builds the Docker image for an ACK service controller. 
 
-Example: $(basename "$0") -q -s ecr
+Example: $(basename "$0") ecr
 
-Options:
-  -s          Provide AWS Service name (ecr, sns, sqs, petstore, bookstore)
-  -i          Controller container image tag (Default: ack-<service>-controller:$VERSION)
-  -q          Build controller container image quietly
-"
+<aws_service> should be an AWS Service name (ecr, sns, sqs, petstore, bookstore)
 
-# Process our input arguments
-while getopts "qs:i:" opt; do
-  case ${opt} in
-    q ) # Build image quietly
-        QUIET=true
-      ;;
-    s ) # AWS Service name
-        AWS_SERVICE=$(echo "${OPTARG}" | tr '[:upper:]' '[:lower:]')
-      ;;
-    i ) # Controller image tag
-        AWS_SERVICE_DOCKER_IMG="${OPTARG}"
-      ;;
-    \? )
-        echo "${USAGE}" 1>&2
-        exit
-      ;;
-  esac
-done
+Environment variables:
+  QUIET:                    Build controller container image quietly (<true|false> Default: false)
+  AWS_SERVICE_DOCKER_IMG:   Controller container image tag (Default: ack-<service>-controller:$VERSION)
```

Comment:
Let's go ahead and place the `Default: XXXX` part on a separate line, similar to how these are done:

https://github.com/aws/aws-controllers-k8s/blob/1ca9215edc0b5883075712e0dbeb7fd5374a8bd0/scripts/helm-package-controller.sh#L24-L27